azure/virtual-machine/diag-settings/win-diag-settings.json

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
Do we need to do this in JSON? Could we do this in HCL and then convert it to JSON?